src/pages/404.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Header from '../components/Header'
+
+function NotFoundPage (): JSX.Element {
+  return (
+    <>
+      <div className='absolute w-full h-screen bg-gray-50 overflow-hidden p-2'>
```

Comment:
I think `h-screen` pushes the page footer out of the viewport.

Reply:
Footer was left out, I have changed from manual usage of header to <Layout />